.github/workflows/rust.yml

```diff
@@ -19,7 +19,8 @@ jobs:
       run: |
         cargo run --example json_to_edn --features "json"
         cargo run --example navigate_edn
-        cargo run --example parse_str
+        cargo run --example from_str_edn
+        cargo run --example from_str_type
```

Comment:
what about `struct_from_str` and `edn_from_str`?

Reply:
Fair enough!

Reply:
Solved!